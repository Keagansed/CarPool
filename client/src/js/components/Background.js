import React, { Component } from 'react';

class Background extends Component {
  //~ constructor(props){
    //~ super(props);
  //~ }

	componentDidMount(){

	}

	render() {
		return (
			<div className="Background">
				<h1>Background</h1>
			</div>
		);
	}
}

export default Background;