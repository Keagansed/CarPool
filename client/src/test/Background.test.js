import React from 'react';
// import ReactDOM from 'react-dom';
import Background from '../js/components/Background';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';


describe('Background Component', () => {
    let container;

    beforeEach(() => {
        container = shallow(<Background />);
    });

    it('captures snapshot', () => {
        const wrapper = renderer.create(<Background />).toJSON();
        expect(wrapper).toMatchSnapshot();
    });

    it('renders correctly', () => {
        expect(container.length).toEqual(1);
    });
});
