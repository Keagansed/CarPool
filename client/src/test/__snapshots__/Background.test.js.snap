// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Background Component captures snapshot 1`] = `
<div
  className="Background"
>
  <h1>
    Background
  </h1>
</div>
`;
